targets/ublk.cpp

```diff
@@ -43,7 +59,27 @@ static int ublksrv_execve_helper(const char *op, const char *type, int argc, cha
 		nenv[0] = ldlp;
 	}
 
-	execve(fp, nargv, nenv);
+
+	if (daemon) {
+		int res;
+
+		setsid();
+		res = fork();
+		if (res == 0) {
+			execve(fp, nargv, nenv);
+		} else if (res > 0) {
+			uint64_t id;
+			int ret = read(evtfd, &id, sizeof(id));
+
+			close(evtfd);
+
+			if (ret == sizeof(id))
+				return list_one_dev(id - 1, false, false);
+		}
+	} else {
+		execve(fp, nargv, nenv);
+	}
+fail:
```

Comment:
Instead of a if (daemon) { } else { }
it could be just be a :

if (!daemon)
      if (execve(...) < 0) {
          fprintf(stderr, ...);
          return errno;
      }
 }
      
At the top and avoiding having that larger codde block being indented.

Reply:
Good point , and the change and reviewed-by tag have been pushed out. 

Thanks for the review!